### PR TITLE
Let the user know to enable DM's from the bot

### LINKED ISSUE #116: Error: Forbidden (Cannot send messages to this user)
https://sentry.dougley.com/dfb/megabot/issues/22/

I would bet this is from ``!stats`` when a user has server DM's blocked.
```
Error: Forbidden (Cannot send messages to this user)
  File "/root/DiscordFeedback/node_modules/superagent/lib/node/index.js", line 698, in Request.callback
    var new_err = new Error(msg);
  File "/root/DiscordFeedback/node_modules/superagent/lib/node/index.js", line 922, in Stream.<anonymous>
    self.callback(null, response);
  File "events.js", line 91, in emitNone
  File "events.js", line 185, in Stream.emit
  File "/root/DiscordFeedback/node_modules/superagent/lib/node/utils.js", line 108, in Unzip.<anonymous>
    stream.emit('end');
...
(5 additional frame(s) were not displayed)

Error: Forbidden (Cannot send messages to this user)
```

### PR DESCRIPTION
**Pull request by Arzte**
Fixes #116

### CODE REVIEW

File: Commands/Commands/utils.js [hunk 1]
```diff
@@ -116,15 +116,24 @@ commands.stats = {
...
           color: 0x59f442,
           title: `${msg.author.username} - Statistics`,
           thumbnail: {
             url: msg.author.staticAvatarURL
           },
           fields: field
-        }).catch(logger.raven) // Send Message to DM error
+        }).catch((e) => {
+          if (e.code === '50007') { // 50007 is the discord json code for not being able to send a DM to a user.
+            msg.reply('Unable to DM you any stats. Maybe you don\'t have DM\'s enabled?').then(errmsg => {
```
Comment (Dougley) [inline Commands/Commands/utils.js]: Use a doublequote string, escaping looks dumb.

File: Commands/Commands/utils.js [hunk 2]
```diff
@@ -116,15 +116,24 @@ commands.stats = {
           inline: true
         }
       )
-      msg.author.openDM().then((e) => {
-        e.sendMessage('', false, {
+      msg.author.openDM().then((dm) => {
+        dm.sendMessage('', false, {
```
Comment (Dougley) [inline Commands/Commands/utils.js:120]: Was renaming this really necessary?

File: Commands/Commands/utils.js (hunk 1 above)
Comment (Arzte) [reply Commands/Commands/utils.js]: 👍

File: Commands/Commands/utils.js (hunk 2 above)
Comment (Arzte) [reply Commands/Commands/utils.js:120]: Since we use `e` for errors, its probably a good idea not to use it elsewhere.

**Review by Dougley — APPROVED** [review]

**Pull request merged**